pkg/test/case.go

```diff
@@ -64,7 +65,14 @@ func (t *Case) DeleteNamespace(ns *namespace) error {
 		return err
 	}
 
-	return cl.Delete(context.TODO(), &corev1.Namespace{
+	ctx := context.Background()
+	if t.Timeout > 0 {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithTimeout(ctx, time.Duration(t.Timeout)*time.Second)
+		defer cancel()
+	}
+
+	return cl.Delete(ctx, &corev1.Namespace{
```

Comment:
If the context is canceled, this will return an error, right?

Reply:
yes... that is correct